engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowInitializationActor.scala

```diff
@@ -63,12 +63,11 @@ case class WorkflowInitializationActor(workflowId: WorkflowId, workflowDescripto
       val backendInitializationActors = Try {
         for {
           (backend, calls) <- workflowDescriptor.backendAssignments.groupBy(_._2).mapValues(_.keys.toSeq)
-          backendConfiguration = BackendConfiguration.backendConfigurationDescriptor(backend).get
           props <- CromwellBackends.shadowBackendLifecycleFactory(backend).map(factory =>
             factory.workflowInitializationActorProps(workflowDescriptor.backendDescriptor, calls)
           ).get
           actor = context.actorOf(props)
-        } yield (actor, backend)
```

Comment:
Yeah, as soon as I started reading the preivous one I realized they were tied together. The "I couldn't find any use" also explains my confusion I believe :)